src/SupervisionSuccess/SupervisionSuccess.js

```diff
@@ -70,7 +71,42 @@ const SupervisionSuccess = () => {
         {
           month: 6,
           baseline: 33600,
-          totalPopulation: 33534.47595,
+          totalPopulation: 33528.56833,
+        },
+        {
+          month: 7,
```

Comment:
I'm a slight bit confused as to why this fixture data is using months instead of years, given that we know we only want to reference years in the chart component. It makes the code a bit more confusing than it needs to be, since we're doing division and modulo arithmetic now, and is a bit misleading.

Edit: Oh I see now that the model in the spreadsheet produces output by month, so you'll get monthly-granularity data but then need to make it yearly in visualization. Got it, that makes sense!